datumaro/components/annotation.py

```diff
@@ -787,3 +789,14 @@ class Caption(Annotation):
 
     _type = AnnotationType.caption
     caption: str = field(converter=str)
+
+
+@attrs(slots=True, order=False)
+class ImageResolution(Annotation):
```

Comment:
I propose to rename it to one of these:

`class SuperResolutionAnnotation(Annotation):`
`class ImageAnnotation(Annotation):`

`ImageResolution` doesn't sound like an annotation type. I'd also expect 2 numbers in that case instead of images.

Reply:
Renamed to `SuperResolutionAnnotation`